docs/core/deploying/native-aot/diagnostics.md

```diff
@@ -1,70 +1,78 @@
 author: lakshanf
 ms.author: lakshanf
 ms.date: 08/07/2023
 ---
-# Diagnostics in native AOT applications
 
-The long-term goal for native AOT application diagnostics is to provide the rich diagnostic experience that developers expect out of a .NET application. The .NET diagnostic experience was built over multiple releases, and a critical part of that journey was customer feedback. Native AOT diagnostics will follow a similar path where the right diagnostic experience is built from prioritized customer feedback in multiple releases.
+# Diagnostics and instrumentation
 
-## Native AOT application features
-
-Native AOT applications have the following characteristics that are important to keep in mind in diagnostic scenarios.
-
-### Trimming as a first-class feature
-
-Unused code in a native AOT application is stripped out of the final binary. Any unbounded reflection used by the application, including its dependent libraries, is trimmed. The native AOT compiler will generate warnings in such cases, and it's critical that these warnings are fixed. For example, if you suppress any warnings without careful analysis, it can lead to hard-to-debug production failures.
-
-### Importance of the symbol file
-
-In native AOT, symbol-file-dependent diagnostics (such as debugging [PerfView](https://github.com/microsoft/perfview) callstacks) don't work at all unless the diagnostic tool has access to the monolithic PDB that was generated when the application was compiled. In .NET, symbol-file-dependent diagnostics generally work just fine, or even great, even if the diagnostic tool has no access to any PDBs that were generated when the application was compiled. That is, symbols for the .NET runtime and the .NET libraries can be pulled from symbol servers, and diagnostic tools can still show function names and accurate call stacks even without access to the compile-time PDBs.
-
-### Application type
-
-Native AOT apps aren't typical managed applications (for example, no JIT). They also aren't typical native applications (for example, they have full GC support). The intent for diagnostics is to meet a reasonable expectation of a product in the .NET family, but the experience should also feel familiar to those using production diagnostics on C++ apps.
+Native AOT shares some, but not all, diagnostics and instrumentation capabilities with CoreCLR. Because of CoreCLR's rich selection of diagnostic utilities, it will sometimes be appropriate to diagnose and debug problems in CoreCLR. Apps which are trim-compatible should not have behavioral differences, so investigations often apply to both runtimes. Nonetheless, some information can only be gathered after publishing, so Native AOT also provides post-publish diagnostic tooling.
 
 ## .NET 8 native AOT diagnostic support
 
 The following table summarizes diagnostic features supported for native AOT deployments:
 
 | Feature | Fully supported | Partially supported | Not supported |
```

Comment:
It would be good to match the feature name in the table with the description headers below for ease of readability. That would require the non-table items to be grouped under a different header.

Reply:
Should be fixed now.